ambassador/ambassador/envoy/v2/v2route.py

```diff
@@ -24,14 +25,31 @@
 
 
 class V2Route(dict):
-    def __init__(self, config: 'V2Config', group: IRHTTPMappingGroup) -> None:
+    def __init__(self, config: 'V2Config', group: IRHTTPMappingGroup, mapping: IRBaseMapping) -> None:
         super().__init__()
 
         envoy_route = EnvoyRoute(group).envoy_route
 
+        mapping_prefix = mapping.get('prefix', None)
+        route_prefix = mapping_prefix if mapping_prefix is not None else group.get('prefix')
+
+        mapping_case_sensitive = mapping.get('case_sensitive', None)
+        case_sensitive = mapping_case_sensitive if mapping_case_sensitive is not None else group.get('case_sensitive', True)
+
+        runtime_fraction = {
+            'default_value': {
+                'numerator': mapping.get('weight', 100),
+                'denominator': 'HUNDRED'
+            }
+        }
+
+        if len(mapping) > 0:
+            runtime_fraction['runtime_key'] = f'routing.traffic_shift.{mapping.cluster.name}'
+
```

Comment:
Hmmm! Does this mean that we can actually look at supporting the runtime settings now?

Reply:
Sure, what kind?